src/tests/users/tokens.test.ts

```diff
@@ -0,0 +1,104 @@
+
+import { authenticateToken, registerToken } from "../../Auth/jwt.tokens";
+import dotenv from "dotenv"
+dotenv.config()
+const jwt = require('jsonwebtoken');
+
+
+describe("User controllers", () => {
+    jest.mock("jsonwebtoken", () => ({
+    sign: jest.fn(),
+    verify: jest.fn()
+}));
+    describe("Getting The user Token", () => {
+        const secretKey = process.env.SECRET_KEY;
+        const options = { expiresIn: '1w' };
+        const expectedToken = `${process.env.expectedToken}`
```

Comment:
Let's use the same format for the environment variable through. 
E.g.: ALL_CAPS_SEPARATED_BY_UNDERSCORES